client-src/elements/chromedash-form-field.js

```diff
@@ -344,4 +405,49 @@ export class ChromedashFormField extends LitElement {
   }
 }
 
+
+/**
+ * Gets the value of a field from a feature entry form, or from the feature.
+ * Looks up the field name in the provided form field values, using the stageOrId
+ * if the field is stage-specific, and returns the corresponding value.
+ * Returns null if not defined or not found.
+ * Handles special cases like shipping milestones and mapped stage fields.
+ * @param {string} fieldName
+ * @param {number|Object|undefined} stageOrId
+ * @param {Array<{name:string, value:*}>} formFieldValues
+ * @return {*}
+ */
+function getFieldValueWithStage(fieldName, stageOrId, formFieldValues) {
+  // Iterate through formFieldValues looking for element with name==fieldName
+  // and stage == stageId, if there is a non-null stageId
+  let stageId;
+  if (typeof stageOrId === 'number') {
```

Comment:
Couldn't we just update all callers to be sure to pass in the stage object in all cases?

Reply:
We don't actually need the stage object if the field is in the current form.  In fact, I don't think we can use the stage object in the case that the start and end OT milestones are in the current form.  But when they are not in the current form, we do need to
look through stage objects to find the max of all OT start milestones.

Reply:
OK, this can be another thing to consider simplifying later.

Reply:
OK thanks.  Yes, I hope we can find some simplifications for all this.